src/layup/utilities/fileAccessUtils.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import os
+import sys
+
+
+def FindFileOrExit(arg_fn, argname):
+    """Checks to see if a file given by a filename exists. If it doesn't,
+    this fails gracefully and exits to the command line.
+
+    Parameters
+    -----------
+    arg_fn : string
+        The filepath/name of the file to be checked.
+
+    argname : string
+        The name of the argument being checked. Used for error message.
+
+    Returns
+    ----------
+    arg_fn : string
+        The filepath/name of the file to be checked.
+
+    """
+
+    logger = logging.getLogger(__name__)
+
+    if os.path.exists(arg_fn):
+        return arg_fn
+    else:
```

Comment:
Current consensus is that we don't want logging, so everywhere there's logging should be printing to stderror or stdout